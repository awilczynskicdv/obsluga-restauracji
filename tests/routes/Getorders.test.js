import { callApi } from "../helpers.js";

const PATH = `api/private/orders`;

test(`[GET] ${PATH} - valid request - should return status 200`, async () => {
  const response = await callApi("get", "/orders/all", {}, {}, false);
 expect(response.data).toMatchSnapshot();
 expect(response.status).toEqual(200);
});
