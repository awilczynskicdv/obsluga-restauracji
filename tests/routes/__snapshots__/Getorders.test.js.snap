// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[GET] api/private/orders - valid request - should return status 200 1`] = `
Object {
  "data": Array [
    Object {
      "created": "30.06.2021",
      "id": 1,
      "price_total": 33.64,
      "status": "preparing",
      "table": "takeaway",
    },
  ],
}
`;
